src/components/battles/animations/shared/Fainting.ts

```diff
@@ -39,18 +39,23 @@ export class Fainting<TGameStartr extends FullScreenPokemon> extends Component<T
             this.gameStarter.physics.getMidY(thing) + thing.height * thing.scale!,
             1,
             (): void => {
+                const playerName = this.gameStarter.itemsHolder.getItem("name");
+                const partyIsWipedText: (string | string[])[][][] = [];
```

Comment:
Let's have a blank line after the group of variable declarations, to keep it visually distinct.